ztp/extra-manifests-builder/03-sctp-machine-config/build.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+MCPROLE=${MCPROLE:-master}
+
+# The SCTP machine config just asks for the sctp kernel module to be loaded
+
+echo "\
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: ${MCPROLE}
+  name: load-sctp-module-${MCPROLE}
```

Comment:
This name change means that any users who have an existing PolicyGenTemplate with the MachineConfig included will experience an update to the MCP. One option would be to name it "load-sctp-module" if mcp==master and load-sctp-module-<mcp> for everything else.

Reply:
I wonder if the contents is identical, if the resulting rendered config will be the same and no extra reboot would be needed...  Going to do some testing.

Reply:
Conclusion: If a 2nd MachineConfig object comes in with identical content and a different name, no extra reboot is incurred.  Both objects show up in the list, and the later one will be ignored.